astroquery/esasky/core.py

```diff
@@ -82,7 +78,7 @@ def list_catalogs(self):
             self._get_catalogs_json(), self.__MISSION_STRING)
 
     def query_object_maps(self, position, missions=__ALL_STRING,
-                          get_query_payload=False, cache=True):
+                          row_limit=DEFAULT_ROW_LIMIT, get_query_payload=False, cache=True):
```

Comment:
This is an API breakage, as the kwargs could be used as positional ones. 

Either please move the new ones to the end, or preferably convert the optional ones to kwarg only. We started doing that with the newer modules, and it may be a good occasion to do it here, too.